src/js/angular/import/directives.js

```diff
@@ -45,3 +45,66 @@ importDirectives.directive('filesOntoLoader', function () {
         }
     };
 });
+
+importDirectives.directive('importResourceTree', importResourceTreeDirective);
+importResourceTreeDirective.$inject = [];
+
+function importResourceTreeDirective() {
+    return {
+        restrict: 'E',
+        templateUrl: 'js/angular/import/templates/import-resource-tree.html',
+        scope: {
+
+            /**
+             * @type {ImportResourceTreeElement}
+             */
+            importResources: '=',
```

Comment:
Rename to just `resources` because these might have not actually been imported yet! Also `importResources` sounds like it's an event/action

Reply:
Done